src/Upgrade/Infrastructure/PackageManager/CommandExecutor/ComposerCommandExecutor.php

```diff
@@ -153,6 +154,6 @@ protected function createResponse(Process $process): ResponseDto
         $output = $process->getExitCode() ? $process->getErrorOutput() : '';
         $outputs = array_filter([$command, $output]);
 
-        return new ResponseDto($process->isSuccessful(), implode(PHP_EOL, $outputs));
+        return new ResponseDto(!$process->getExitCode(), implode(PHP_EOL, $outputs));
```

Comment:
Why this change?

Reply:
@vol4onok , I also don't understand why have you changed it here. `isSuccessful` already checks the exit code: 

```
public function isSuccessful()
{
    return 0 === $this->getExitCode();
}
```

When we reach the timeout, exit code will be null (if I understand it correct) and isSuccessful() will return false.

Reply:
Also more clear for humans :)

Reply:
reverted